cmd/tid/main.go

```diff
@@ -13,29 +13,41 @@ import (
 	boltdb "github.com/boltdb/bolt"
 
 	_ "github.com/SeerUK/tid/pkg/state/migrate/versions"
+	"github.com/SeerUK/tid/pkg/toml"
+	"github.com/SeerUK/tid/pkg/types"
 )
 
 func main() {
-	db := getBoltDB()
+	dir, err := tid.GetLocalDirectory()
+	fatal(err)
+
+	tomlConfig := getTomlConfig(dir)
+
+	db := getBoltDB(dir)
 	defer db.Close()
 
 	backend := bolt.NewBoltBackend(db)
 
 	// Initialise the backend, preparing it for use, ensuring it's up-to-date.
-	err := migrate.Backend(backend)
-	fatal(err)
+	migErr := migrate.Backend(backend)
```

Comment:
I'm thinking this variable name was changed to `migErr` because of a compiler error saying the variable was already defined? if this is the case, you can use `=` instead of `:=` to assign to an already created variable